app/presenters/nsm/cost_summary/report.rb

```diff
@@ -22,7 +23,7 @@ def sections
           {
             card: {
               title: data.title,
-              actions: actions(name)
+              actions: actions(data.change_key || name)
```

Comment:
See above comment for how to update this

Reply:
updated